test/testcafe/framework/page-objects/BasePageObject.js

```diff
@@ -1,11 +1,13 @@
 import BaseFormObject from './components/BaseFormObject';
-import { Selector, ClientFunction } from 'testcafe';
+import { Selector, ClientFunction, userVariables } from 'testcafe';
+
+const BACK_TO_SIGNIN_LINK = '[data-se="cancel"]';
 
 const SIGNOUT_LINK = '.auth-footer .js-cancel';
 const GO_BACK_LINK = '.auth-footer .js-go-back';
 const SKIP_LINK = '.auth-footer .js-skip';
 const SKIP_SET_UP_LINK = '.auth-footer .js-skip-setup';
-const SWITCH_AUTHENTICATOR_LINK = '.auth-footer .js-switchAuthenticator';
+const SWITCH_AUTHENTICATOR_LINK = '[data-se="switchAuthenticator"]';
```

Comment:
Confirmed in LinkUtil.js (v2) that the switch authenticator links (in either enroll or verify) both include this data-se attribute.

Reply:
Is it possible to use find by role when the name can be 1 of two possible options? 

In this case, the link text can either be `Return to authenticator list` or `Verify with something else` I can create two separate functions, but it will require me to update the test to use the correct function (since the text varies based on if you are enrolling or verifying).